src/components/Maps/Maps.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <v-carousel>
+    <v-carousel-item v-for="(item,i) in items" :key="i" :src="item.src" contain="true"></v-carousel-item>
+  </v-carousel>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      items: [
+        {
+          src: require("../../../public/pics/maps/Barovia/baroviaregion.png")
```

Comment:
There is no need to require static resources (from 'public' folder), just 'pics/maps/Barovia/baroviaregion.png' will be enough